tests/test_gpt4all_chat_actors.py

```diff
@@ -73,25 +91,26 @@ def test_extract_question_with_date_filter_from_relative_year(loaded_model):
 @pytest.mark.chatquality
 def test_extract_multiple_explicit_questions_from_message(loaded_model):
     # Act
-    response = extract_questions_falcon("What is the Sun? What is the Moon?", loaded_model=loaded_model)
+    response = extract_questions_llama("What is the Sun? What is the Moon?", loaded_model=loaded_model)
 
     # Assert
     expected_responses = ["What is the Sun?", "What is the Moon?"]
-    assert len(response) == 2
-    assert expected_responses == response
+    assert len(response) >= 2
+    assert expected_responses[0] == response[-2]
+    assert expected_responses[1] == response[-1]
 
 
 # ----------------------------------------------------------------------------------------------------
 @pytest.mark.chatquality
 def test_extract_multiple_implicit_questions_from_message(loaded_model):
     # Act
-    response = extract_questions_falcon("Is Morpheus taller than Neo?", loaded_model=loaded_model, run_extraction=True)
+    response = extract_questions_llama("Is Morpheus taller than Neo?", loaded_model=loaded_model)
 
     # Assert
     expected_responses = [
-        ("morpheus", "neo"),
+        ("morpheus", "neo", "height", "taller", "shorter"),
     ]
-    assert len(response) == 2
+    assert len(response) == 3
```

Comment:
Shouldn't this extract just two questions to search for in the knowledge base and arrive at final answer?
1. What is Morpheus's height?
2. What is Neo's height?

Reply:
To compensate for the model's unreliability, I have it always returning the user's original question in the list of extracted questions. so the results will always be `1 + len(extracted_questions)`.